pkg/webhook/common.go

```diff
@@ -20,6 +20,8 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
 )
 
+type requestResponse string
```

Comment:
nit: should we have different types per package if we are declaring the constants elsewhere?

I'm also fine if we consolidate all the response constants. It just seems a bit weird to declare a trivial type and not see it used immediately.

Reply:
Done